packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNode.ts

```diff
@@ -159,7 +160,18 @@ export class SummarizerNode implements IRootSummarizerNode {
 			}
 		}
 
-		const result = await this.summarizeInternalFn(fullTree, true, telemetryContext);
+		const incrementalContext: IIncrementalContext = {
+			sequenceNumber: this.wipReferenceSequenceNumber ?? this._changeSequenceNumber,
+			previousSequenceNumber: this._latestSummary?.referenceSequenceNumber ?? 0,
+			parentPath: this._latestSummary?.fullPath.path ?? "",
```

Comment:
What's the parent path used for? Using "" seem wrong. How is "" supposed to be used?

Reply:
Good point, let me investigate this.